app/views/visits/_form.html.haml

```diff
@@ -43,6 +43,9 @@
           .form-group.col-4.mb-0
             = f.label :effort_billing_qty, class: 'required'
             = f.text_field :effort_billing_qty, class: 'form-control'
+          .form-group.col-4{ id: 'select-t-qty' }
+            = f.label t(:calendars)[:pppv][:visits][:select_all_t]
+            = f.check_box :set_all_insurance
```

Comment:
Can you add the Bootstrap 4 styles for this checkbox for consistency? We can also make the checkbox a bit bigger so that it's more prominent by using the custom checkbox element styles (see the documentation below) and adding `custom-control-lg` to the `custom-control` element, which we do in a few places in the application for larger checkboxes.

https://getbootstrap.com/docs/4.5/components/forms/#checkboxes

Reply:
I'm going to leave the checkbox as is.